src/routes/board/components/log/Log.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+import classes from "./Log.module.css";
+import withLogProps from "./withLogProps";
+
+const Log = ({ list, id, logText, listEndRef }) => (
+  <div className={classes.container}>
+    {list.map((item) => (
+      <p key={id(item)} className={classes.text}>
+        {logText(item)}
+      </p>
+    ))}
+
+    <div ref={listEndRef} />
```

Comment:
What does this do?

Reply:
Scrolls to bottom of Log component, changed the logic in HOC and removed it.